metashare/api/sf_run_flow.py

```diff
@@ -265,62 +267,61 @@ def run_flow(*, repo_owner, repo_name, repo_branch, user, flow_name):
-        get_access_token(
-            login_url=login_url,
-            org_result=org_result,
-            scratch_org_config=scratch_org_config,
-            sf_client_id=sf_client_id,
-            sf_client_secret=sf_client_secret,
-        )
-        org_config = save_settings(
-            cci=cci,
-            login_url=login_url,
-            org_name=org_name,
-            scratch_org_config=scratch_org_config,
-            scratch_org_definition=scratch_org_definition,
-            sf_client_id=sf_client_id,
-        )
-
-        # ---
-        # Scratch org construction is done but we haven't run a flow
-        # yet.  This is the point where you would want to serialize
-        # scratch_org_config.config to store in the database for use
-        # later. Then reconstitute by running construc_org_config
-        # ---
-
-        if org_config:
-            # Run flow (takes care of getting a new access token)
-            flow = cci.get_flow(flow_name)
-            with cd(project_path):
-                flow.run(org_config)
+    cci = BaseCumulusCI(
+        repo_info={
+            "root": project_path,
+            "url": repo_url,
+            "name": repo_name,
+            "owner": repo_owner,
+            "commit": repo_branch,
+        }
+    )
+    devhub_api = get_devhub_api(
+        devhub_username=devhub_username, sf_client_id=sf_client_id
+    )
+    scratch_org_config, scratch_org_definition = get_org_details(
+        cci=cci, org_name=org_name, project_path=project_path
+    )
+    org_result = get_org_result(
+        email=email,
+        repo_owner=repo_owner,
+        repo_name=repo_name,
+        repo_branch=repo_branch,
+        scratch_org_config=scratch_org_config,
+        scratch_org_definition=scratch_org_definition,
+        cci=cci,
+        devhub_api=devhub_api,
+        sf_client_id=sf_client_id,
+    )
+    mutate_scratch_org(
+        scratch_org_config=scratch_org_config, org_result=org_result, email=email
+    )
+    login_url = get_login_url(org_result)
+    get_access_token(
+        login_url=login_url,
+        org_result=org_result,
+        scratch_org_config=scratch_org_config,
+        sf_client_id=sf_client_id,
+        sf_client_secret=sf_client_secret,
+    )
+    org_config = deploy_org_settings(
+        cci=cci,
+        login_url=login_url,
+        org_config=scratch_org_config,
+        org_name=org_name,
+        scratch_org_config=scratch_org_config,
+        scratch_org_definition=scratch_org_definition,
+        sf_client_id=sf_client_id,
+    )
+
+    # ---
+    # Scratch org construction is done but we haven't run a flow
+    # yet. This is the point where you would want to serialize
+    # scratch_org_config.config to store in the database for use
+    # later. Then reconstitute by running construc_org_config
+    # ---
+
+    # Run flow (takes care of getting a new access token)
+    flow = cci.get_flow(flow_name)
+    with cd(project_path):
+        flow.run(org_config)
```

Comment:
Missing a `return` statement?

Reply:
Doesn't actually return anything.

Reply:
Yeah, I just figured that out and pushed the fix :)

Reply:
Right... but I think that's the problem? It's being called as if it returns `org_config`: https://github.com/SFDO-Tooling/MetaShare/pull/142/files#diff-e148a690b7578d0eded316df5a9c8939R115